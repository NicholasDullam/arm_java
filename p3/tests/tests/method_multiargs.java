class Base{
    public static int method(int a, int b) {
        System.out.print(a);
        System.out.println(b);
        int x = a + b;
        return x;
    }
	public static void main(String[] args){
        System.out.println(args[0]);
        int a = method(7, 3);
        System.out.println(a);
	}
}
