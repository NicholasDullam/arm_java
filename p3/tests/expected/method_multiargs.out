745
73
10
